content/BatchRequestContentCollection.md

```diff
@@ -0,0 +1,23 @@
+# BatchRequestContentCollection
+
+## Objectives
+
+Provide a class that does not have limitation of that 20 requests by creating new `BatchRequestContent` objects to hold new requests to simplify the batching for advanced scenarios. 
+The `BatchRequestContentCollection` is essentially a collection wrapper for `BatchRequestContent` and is not expected to be serializable as the individual `BatchRequestContent` instance are sent out independently and will receive independent responses.
+
+### Requirements
+
+Similar to [BatchRequestContent](BatchRequestContent.md), it must support the below functionality while holding a collection of BatchRequestContent instances.
+
+- It MUST be possible to add native platform HTTP request objects to a BatchRequestContent object. The request URL MUST be a relative URL. Adding Graph specific Request objects MAY be supported.
+- Added requests MUST be assigned unique identifiers. Those identifiers MAY be user provided or autogenerated. The uniqueness SHOULD be enforced/validated across the same `BatchRequestContent` instance the request objects are held in AND across the `BatchRequestContent` instances.
+- It MUST be possible to create dependencies between request objects. However, these dependencies SHOULD be enforced/validated across the same `BatchRequestContent` instance the request objects are held in and not across `BatchRequestContent` instances.
+
+> Given the similar API experience between `BatchRequestContentCollection` and `BatchRequestContent`, `BatchRequestContent` SHOULD be used internally and not exposed publicly to avoid confusion for the end user. Adding new requests should automatically generate new `BatchRequestContent` instances to the `BatchRequestContentCollection`.
+
+### Performance Considerations
+
+The request client MAY send `BatchRequestContent` instances in parallel if the `BatchRequestContentCollection` contains more than one `BatchRequestContent` instance
```

Comment:
What is the impact on potential throttling here? Are we putting the customers at risk of hitting the limits earlier?

Reply:
I've clarified this to be an optional opt-in behavior that should be done in serial by default. So as to minimise risk of throttling.